submissions/opeyemiPeter/database/transportation-company.sql

```diff
@@ -45,3 +44,29 @@ CREATE TABLE accidentCases(
 SELECT * FROM accidentCases;
 
 
+-- Task 3 Transportation Data Query.
+
+# 1. Total number of passengers who survived
+	select count(*) from accidentCases where accidentStatus > 0;
+	-- Answers => 342
+
+# 2. total number of unsurvived passengers
+  SELECT COUNT(*) FROM accidentCases WHERE accidentStatus = 0;
+  -- Answers => 549
+
+# 3. Name and sex of passengers under the age of 27 that embarked at Queenstown and Cherbourg
+	SELECT registeredPassengers.fullName, registeredPassengers.sex from registeredPassengers
+	inner join tripDetails on passengerId =tripDetails.passengerId where age  < '27'and embarkationPoint 
+    like "Q%" or embarkationPoint like "C%";
+    
+# 4. Number of passengers who embarked at Southampton and survived
+	SELECT count(*) from accidentCases left join tripdetails on accidentcases.id = tripdetails.id where accidentStatus > 0 and embarkationPoint = 'S';
+    -- Answers => 218;
+
+    
+   # 5. ID, Name and Total Number of passengers who paid a fare greater than $100 and above the age of 35 had siblings or spouses on board 
+    SELECT COUNT(*) FROM registeredpassengers
+	LEFT JOIN tripdetails ON registeredpassengers.id = tripdetails.id
+	WHERE tripFare > 100 AND age > 35 AND numberOfSiblingsSpouse > 0; 
+	
+    -- Answers => 7;
```

Comment:
The total number of passengers is not 7

Reply:
@adaugochi, I have tried working on this. but I am still getting the same answer  7. 
for the record: the total number of data on all the imported tables is 891, peradventure there is an update on the CSV file i imported

Reply:
@adaugochi kindly Review

Reply:
Ok, I will check this again

Reply:
![image](https://user-images.githubusercontent.com/63324192/108078290-55771a80-7022-11eb-8006-1426114b19f1.png)
![image](https://user-images.githubusercontent.com/63324192/108078321-6162dc80-7022-11eb-934a-935774f3a20e.png)